### PR TITLE
chore: add initial database migrations

### PR DESCRIPTION
**Pull request by dilyara4949**
Added initial database migration

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+create-migration:
+	@read -p "migration name: " name; \
+	migrate create -ext sql -dir internal/database/postgres/migration -seq $$name
+
+migrate-up:
+	migrate -database $(DB_URL) -path internal/database/postgres/migration up
+
+migrate-down:
+	migrate -database $(DB_URL) -path internal/database/postgres/migration down
```
Comment (MihailGorelikov) [inline Makefile:11]: need to add empty line at the EOF

File: internal/database/postgres/migration/000001_init-tables.up.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
+CREATE TABLE roles (
+                      id UUID PRIMARY KEY,
+                      role VARCHAR UNIQUE NOT NULL
```
Comment (MihailGorelikov) [inline internal/database/postgres/migration/000001_init-tables.up.sql:3]: if table names roles, better to name as name

**Review by MihailGorelikov — CHANGES_REQUESTED** [review]

File: internal/database/postgres/migration/000001_init-tables.up.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
+CREATE TABLE roles (
+                      name VARCHAR PRIMARY KEY
```
Comment (MihailGorelikov) [inline internal/database/postgres/migration/000001_init-tables.up.sql:2]: why did you removed id field?

File: internal/database/postgres/migration/000001_init-tables.up.sql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                        created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                        updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                        FOREIGN KEY (flight_id) REFERENCES Flights(id),
+                        FOREIGN KEY (user_id) REFERENCES Users(id),
+                        FOREIGN KEY (rank) REFERENCES Ranks(name)
+);
+
+
+insert into roles (name) values ('user'), ('admin');
+insert into ranks (name) values ('economy'), ('business'), ('deluxe')
```
Comment (MihailGorelikov) [inline internal/database/postgres/migration/000001_init-tables.up.sql:49]: SQL statements should be CAPITALISED

Comment (MihailGorelikov) [inline internal/database/postgres/migration/000001_init-tables.up.sql:49]: missing empty line at EOF

**Review by MihailGorelikov — CHANGES_REQUESTED** [review]

File: internal/database/postgres/migration/000001_init-tables.up.sql (hunk 3 above)
Comment (dilyara4949) [reply internal/database/postgres/migration/000001_init-tables.up.sql:2]: I thought it would be easier to work with

**Review by MihailGorelikov — APPROVED** [review]
LGTM

**Pull request merged**